config/cesm/machines/config_machines.xml

```diff
@@ -671,9 +671,12 @@ This allows using a different mpirun command to launch unit tests
     <mpirun mpilib="default">
       <executable>mpiexec</executable>
       <arguments>
-        <arg name="label"> --label</arg>
-        <arg name="buffer"> --line-buffer</arg>
-        <arg name="num_tasks" > -n {{ total_tasks }}</arg>
+        <arg name="label">--label</arg>
```

Comment:
This change is for cesm2.2 - we have added an mpibind script provided by cisl [here.](https://github.com/ESMCI/ccs_config_cesm/blob/main/machines/derecho/config_machines.xml#L24)    Can you compare and contrast these two methods so that we can move forward with a consistant approach?

Reply:
I was not aware of the `mpibind` script.  I will give that a try.